### ISSUE TITLE
Need help to build UPBGE Eevee Branch on Linux

### ISSUE THREAD

**Issue by chicortiz** [UPBGE/upbge#1053]
Hello folks!

Could anybody give me detailed instructions on how to build Upge Eevee Branch for Linux?

Thank you very much!

Ortiz

**Comment by youle31** [comment]
- You can clone the upbge sources:
git clone https://github.com/UPBGE/blender.git

- In source\.git\modules\release\scripts\addons
you can edit config file and replace the url with this:
url = git://git.blender.org/blender-addons.git

- Then you have to checkout eevee_merge8 branch (from a terminal opened in the sources folder):
git checkout eevee_merge8

- Update/Init the submodules:
git submodule update --init --recursive
git submodule foreach git checkout master
git submodule foreach git pull --rebase origin master

- Update the branch:
git pull --rebase

- Update the libraries:

In sources\build_files\build_environment, run install_deps.sh script:
sudo ./install_deps.sh --with-all

- Install cmake from your synaptic package manager (and cmake-gui)

- Run cmake and choose the sources folder and a destination folder (which will be created)

- Click 1 time on configure, and you will have a red list of options to build or not to build:

Disable:
WITH_FREESTYLE
WITH_PLAYER (it doesn't work on my linux mint)
you can also disable WITH_CYCLES if it is causing issues
Enable:
WITH_OPENSUBDIV

- Click again on configure now (the list should turn white)
- Click on generate (Take care that there is no error message)

- Go in your destination folder:
open a terminal and type:
make -j5 (if you have 4 cores on your cpu it is j5, if you have 2 cores, j3, 6 cores, j7)

if the compilation succeed:
make install

the blender binary (executable) should be in destinationfolder/bin

If you have issues to compile, you can try to disable things in cmake, and you can have a look at blender guide https://wiki.blender.org/wiki/Building_Blender/Linux and troobleshooting sections...

Only embedded works. It is very experimental, and you might have even more issues on linux than on windows. Good luck

**Comment by chicortiz** [comment]
Thanks a lot @youle31 !

**Comment by youle31** [comment]
In lasts version of lordloki branch you should be abled to compile with make from sources directory:

update deps running ./install_deps.sh script

then in sources folder:

make update
make -j7

Note: This is building WITH_CYCLES but without OPENCOLORIO and this is causing color management issues in cycles, but it would be an issue to fix on BF Blender side. I'll maybe open a bug report in BF tracker if I have a fix for this

**Comment by SebastianMestre** [comment]
Hello, I have downloaded v0.2.4 for linux from the experimental downloads page, but it seems like there are no render attachments?

Looking through the commit log, it looks like render attachments were merged to master on December 2018 (23183936242afafaecd8a00a116668fa68e8381c), not sure if this is part of release 0.2.4 though...

After asking on the forums, I found out there should be an Attachments panel under the render tab of the properties editor, but it's just not there. Is there a build available that has this? Or will I have to build it myself?

Thank you
